test/interop/hyperlane_test.go

```diff
@@ -124,6 +125,25 @@ func (s *HyperlaneTestSuite) TestHyperlaneTransfer() {
 	s.Require().Equal(math.NewInt(1000).Int64(), balance.Amount.Int64())
 }
 
+func (s *HyperlaneTestSuite) TestSyntheticTokensDisabled() {
+	const (
+		SimappDomainID = 1337
+	)
+
+	ismIDSimapp := s.SetupNoopISM(s.simapp)
+	mailboxIDSimapp := s.SetupMailBox(s.simapp, ismIDSimapp, SimappDomainID)
+
+	msgCreateSyntheticToken := warptypes.MsgCreateSyntheticToken{
+		Owner:         s.celestia.SenderAccount.GetAddress().String(),
+		OriginMailbox: mailboxIDSimapp,
+	}
+
+	res, err := s.celestia.SendMsgs(&msgCreateSyntheticToken)
+	s.Require().Error(err)
+	s.Require().ErrorContains(err, "module disabled synthetic tokens")
```

Comment:
can't use `ErrorIs(err, target)` because they don't register sdk errors with codes, but this is fine